script/helperFunctions.sh

```diff
@@ -1977,6 +1977,37 @@ function getAllNetworksArray() {
   # return ARRAY
   printf '%s\n' "${ARRAY[@]}"
 }
+
+# Function to retrieve coreFacets from global.json
+function getCoreFacetsArray() {
+  # Ensure GLOBAL_FILE_PATH is set and not empty
+  if [[ -z "$GLOBAL_FILE_PATH" ]]; then
+    echo "Error: GLOBAL_FILE_PATH is not set or empty."
+    exit 1
+  fi
+
+  local FILE="$GLOBAL_FILE_PATH"
```

Comment:
Is there a need to introduce a new variable if this has the same value as the global variable and is never written to?